client/app/html/index.html

```diff
@@ -5,6 +5,7 @@
     <title>Weave Scope</title>
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script language="javascript">window.__WEAVEWORKS_CSRF_TOKEN = "$__CSRF_TOKEN_PLACEHOLDER__";</script>
```

Comment:
Don't quite understand where `$__CSRF_TOKEN_PLACEHOLDER__` gets replaced with the actual token, but could we maybe make this string a constant since it's used in two places on frontend?